core/templates/dev/head/services/context.service.spec.ts

```diff
@@ -41,6 +41,9 @@ describe('Context service', function() {
         $provide.value('UrlService', {
           getPathname: function() {
             return '/explore/123';
+          },
+          getHash: function() {
+            return ''
```

Comment:
Missing semicolon at end.

/cc @oppia/dev-workflow-team This should be an eslint check?

Reply:
Done